src/GitHub.VisualStudio.UI/Resources.resx

```diff
@@ -360,8 +360,8 @@
   <data name="CompareFileAsDefaultAction" xml:space="preserve">
     <value>Compare File as Default Action</value>
   </data>
-  <data name="OpenFile" xml:space="preserve">
-    <value>Open File</value>
+  <data name="ViewFile" xml:space="preserve">
+    <value>View File</value>
```

Comment:
Users could easily miss the fact that the "Working Directory" commands are writable. 
Maybe change this label to, "View File (read-only)"?